packages/esm-service-queues-app/src/active-visits/active-visits-table.test.tsx

```diff
@@ -114,8 +114,8 @@ describe('ActiveVisitsTable: ', () => {
     });
 
     const expectedTableRows = [
-      /Eric Test Ric Not Urgent Waiting For Triage 206 hours and 2 minutes/,
-      /John Smith Emergency Attending Clinical Consultation 206 hours and 2 minutes/,
```

Comment:
Still not really sure about this as a complete solution.  I agree about removing the hard-coded text, but I'd be inclined to then just split the service and status into 2 separate columns, rather than try to merge them together into a pseudo-status.

Regardless, the function that calls the buildStatusString method should no longer pass in a status that is converted to lower case, but just preserve whatever case is in the status display.

Reply:
@mseaton , I am not sure what you are suggesting here. As you could see below, the tests are passing fine now after I made this change to the buildStatusString. 
Are you suggesting we change the queue display table to add an extra column to display the service? This won't be helpful at all for the MCOE EMR.

Reply:
Probably easier to discuss @cioan if it's unclear what I'm suggesting

Reply:
Thanks @mseaton for clarifying all those issues. Based on our discussion, I have updated the PR with adding the "dash" between the Status and Service Name to differentiate between the two different columns. I also added two additional tickets to document the other 2 issues we discovered:
- https://openmrs.atlassian.net/browse/O3-2698
- https://openmrs.atlassian.net/browse/O3-2699